dotcom-rendering/src/web/browser/islands/doHydration.tsx

```diff
@@ -39,7 +39,9 @@ export const doHydration = async (
 		.then((module) => {
 			/** The duration of importing the module for this island */
 			const importDuration = importEnd();
-			const clientOnly = element.getAttribute('clientOnly') === 'true';
```

Comment:
The actual attribute name is `clientonly`, I don't case actually matters, but I've updated this to use lowercase just in case.